example/go.mod

```diff
@@ -0,0 +1,14 @@
+module example
+
+go 1.22.6
+
+toolchain go1.22.8
+
+require github.com/blocky/basm-go-sdk v0.0.0
+
+require (
+	github.com/josharian/intern v1.0.0 // indirect
+	github.com/mailru/easyjson v0.9.0 // indirect
+)
+
+replace github.com/blocky/basm-go-sdk => ../
```

Comment:
Forcing example to use the version of the SDK from the TLD, not from the latest commit, which is always at least one behind